Makefile

```diff
@@ -0,0 +1,271 @@
+PROJECT_NAME=devopsconsole-operator
+PACKAGE_NAME:=github.com/fabric8-services/$(PROJECT_NAME)
+CUR_DIR=$(shell pwd)
+TMP_PATH=$(CUR_DIR)/tmp
+INSTALL_PREFIX=$(CUR_DIR)/bin
+VENDOR_DIR=vendor
+INCLUDE_DIR=make
+SOURCE_DIR ?= .
+SOURCES := $(shell find $(SOURCE_DIR) -path $(SOURCE_DIR)/vendor -prune -o -name '*.go' -print)
+
+ifeq ($(OS),Windows_NT)
+	include ./$(INCLUDE_DIR)/Makefile.win
```

Comment:
Please, let's ignore Windows, okay? Nobody can test this code at the moment.

Reply:
@alexeykazakov @dipak-pawar I sort of agree with @kwk. Was there a specific reason to have Windows support in the toolchain operator? We should be consistent here.

Reply:
there is  no specific reason, also it was not tested by anyone, we can remove this. it was copy pasted when I wrote makefile, but we can get rid out of that

Reply:
I'm fine with removing Windows for now.

Reply:
Done. 98d2b3f